Tests/Resources/config.php

```diff
@@ -15,7 +15,7 @@
         'base_url' => 'http://localhost',
         'sahi' => array(),
         'zombie' => array(),
-        'selenium' => array(),
+//        'selenium' => array(),
```

Comment:
Yeah, the `Selenium1` driver is deprecated now, but removing as part of config isn't directly related to this PR.